include/wx/withimages.h

```diff
@@ -65,7 +65,7 @@ class WXDLLIMPEXP_CORE wxWithImages
     //
     // Override OnImagesChanged() in the derived class to update the actually
     // shown images.
-    void SetImages(const Images& images)
+    virtual void SetImages(const Images& images)
```

Comment:
This contradicts the comment just above and shouldn't be done.
```suggestion
    void SetImages(const Images& images)
```